indico/modules/categories/controllers/display.py

```diff
@@ -138,8 +138,20 @@ def _serialize_category(category, with_path=False, with_favorite=False):
         'can_access': category.can_access(session.user),
     }
     if with_path:
-        data['path'] = [{'id': c.id, 'title': c.title}
-                        for c in category.parent_chain_query.options(load_only('id', 'title'))]
+        if child_path:
+            data['path'] = child_path[:]
+            for segment in reversed(child_path):
+                data['path'].pop(-1)
```

Comment:
but you are not using the data returned from `pop` so i think using `del` instead is a good idea